src/panels/lovelace/editor/hui-glance-card-editor.ts

```diff
@@ -35,37 +45,122 @@ export class HuiGlanceCardEditor extends hassLocalizeLitMixin(LitElement)
     }
 
     return html`
+      ${this.renderStyle()}
       <paper-input
         label="Title"
         value="${this._config!.title}"
         .configValue=${"title"}
+        .type=${"input"}
+        @value-changed="${this._valueChanged}"
+      ></paper-input>
+      <hui-theme-select-editor
+        .hass=${this.hass}
+        .value=${this._config!.theme}
+        .type=${"input"}
+        .configValue=${"theme"}
+        @change=${this._valueChanged}
+      ></hui-theme-select-editor><br>
+      <paper-input
+        label="Columns"
+        value="${this._config!.columns || ""}"
+        .configValue=${"columns"}
+        .type=${"input"}
         @value-changed="${this._valueChanged}"
-      ></paper-input><br>
+      ></paper-input>
+      <h4>Entities</h4>
+      <div class="entities">
+        ${this._configEntities!.map((entityConf, index) => {
+          return this.renderEntity(entityConf, index!);
+        })}
+      </div><br>
+      <paper-button
+        noink
+        raised
+        @click=${this.addEntity}
+      >Add Entity</paper-button>
+      <br><br>
       <paper-checkbox
         ?checked="${this._config!.show_name !== false}"
         .configValue=${"show_name"}
+        .type=${"checkbox"}
         @change="${this._valueChanged}"
       >Show Entity's Name?</paper-checkbox><br><br>
       <paper-checkbox
         ?checked="${this._config!.show_state !== false}"
         .configValue=${"show_state"}
+        .type=${"checkbox"}
         @change="${this._valueChanged}"
       >Show Entity's State Text?</paper-checkbox><br>
     `;
   }
 
+  private renderStyle(): TemplateResult {
+    return html`
+      <style>
+        .entities {
+          padding-left: 20px;
+        }
+      </style>
+    `;
+  }
+
+  private renderEntity(
+    entityConf: EntityConfig,
+    index: number
+  ): TemplateResult {
+    return html`
+      <ha-entity-picker
+        .hass="${this.hass}"
+        .value="${entityConf.entity || entityConf}"
+        .type=${"entity"}
+        .index="${index}"
+        @change="${this._valueChanged}"
+      ></ha-entity-picker>
+    `;
+  }
+
+  private addEntity() {
+    const newConfig = this._config!;
+    const newConfigEntites = this._configEntities!;
+
+    newConfigEntites.push(newConfigEntites[0].entity);
```

Comment:
I would like to each add a blank entity or an unused entity. But a blank entity gets an error in Process Entities for Unknown EntityID or bad EntityID. And Compute entities require a config Parameter that contains all the views currently active

Reply:
Let's not use those methods.

Reply:
Actually, why would you add a blank entity row, the moment a user picks an entity, you push a row to the stack which then has an entity picked?

Also, always push an object, as that makes it possible to edit the other options.

Reply:
I only added this so the functionality would work and commented on it as I knew it needed changed but I dont understand 

> the moment a user picks an entity, you push a row to the stack which then has an entity picked?

When the user presses add I wanted to go through the unused entities and add one from there and then they could change it. The only other alternative I thought from that was a blank entity so the placeholder is just entity and they start fresh

Reply:
What do you think is the best way to add a new entity row? User clicks `Add Entity` do they get a empty select box or a auto filled entity?